pkg/controller/operandrequest/reconcile_operand.go

```diff
@@ -104,7 +104,6 @@ func (r *ReconcileOperandRequest) reconcileOperand(requestInstance *operatorv1al
 			if csv.Status.Phase != olmv1alpha1.CSVPhaseSucceeded {
 				klog.Errorf("The ClusterServiceVersion for Subscription %s is not Ready", opdConfig.Name)
 				requestInstance.SetMemberStatus(operand.Name, operatorv1alpha1.OperatorInstalling, "")
-				continue
```

Comment:
@DanielXLee I remove this continue is because of ibm-iam-operator, which CSV won't be in the succeeded status until its dependency operands are deployed.

Reply:
I think we need to create an issue for iam update their operator don't depend on other operators in their operator deployment.

Reply:
okay, remove it at first, after IAM operator enhanced. add it back again